A/ASL/build_tarballs.jl

```diff
@@ -16,6 +16,8 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/solvers/
 mkdir -p ${libdir}
+incdir=${prefix}/include/ASL
```

Comment:
Where are the headers usually installed?  In `include/ASL`?  We should follow the standard.  CoinUtils was looking for `getstub.h`, not `ASL/getstub.h`, so I think we should install to `${prefix}/include`

Reply:
This library doesn't install them anywhere. The AMPL-MP installs them in `include/asl`. It's easy to pass flags to configure scripts to help them find the headers, but I'm happy to put them in `include` if you prefer.

Reply:
We should install where developers expect them.  Ideally, we should use at most
```
-I${prefix}/include
```
when linking to this library.